theia-extensions/viewer-prototype/src/browser/trace-viewer/trace-viewer-toolbar-commands.ts

```diff
@@ -0,0 +1,11 @@
+import { Command, MenuPath } from '@theia/core';
+
+export const TRACE_VIEWER_TOOLBAR_COMMAND_FILTER: Command = {
+    id: 'trace.viewer.toolbar.filter',
+    label: 'Trace Viewer Toolbar Filter',
+    iconClass: 'fa fa-filter',
```

Comment:
Just checking, is this correct? I'm not familiar with this 'fa fa' thing...

Reply:
Yes this is correct. 'fa fa' is font awesome thing :)